tools/depcheck/README.md

```diff
@@ -0,0 +1,176 @@
+depcheck
+========
+
+Dependency analysis tool for Go repositories.
+
+### Why
```

Comment:
`## Why`

Reply:
Generally I'd like to see the following structure:

TL;DR
1. Why
2. Getting started
  * Compiling
  * Running (put examples here)
  * Limitations
3. Advanced usage
  * Analyze (put examples here)
  * Trace (put examples here)
  * Filters
  * Excludes

Reply:
Thanks, will update overall structure of this doc